.github/workflows/test.yml

```diff
@@ -17,9 +17,7 @@ jobs:
     - uses: actions/checkout@v3
 
     - name: Install pnpm
-      uses: pnpm/action-setup@v2
-      with:
-        version: 8.5.0
+      uses: pnpm/action-setup@v4
```

Comment:
This was a one-line change, so I bumped it while I was deleted the unnecessary version parameter (it'll read from `package.json`)